src/components/MapControls/MapControls.js

```diff
@@ -40,13 +43,102 @@ const degreesToRadians = (degrees) => {
   return degrees * (Math.PI / 180);
 };
 
+const useStyles = makeStyles((theme) => {
+  return {
+    mapControls: {
+      position: "absolute",
+      display: "flex",
+      flexDirection: "column",
+      gap: 8,
+      right: (props) => (props.isMobile ? 15 : props.overlayWidth + 15),
+      "& .rs-zooms-bar": {
+        "& .ol-zoomslider": {
+          border: "1px solid #5a5a5a !important",
+          paddingBottom: "3px !important",
+        },
+        "& .ol-zoomslider-thumb": {
+          background: `${theme.colors.darkGray} !important`,
+        },
+        "& .rs-zoom-in, .rs-zoom-out": {
+          margin: "10px 0",
+          background: "white",
+          boxShadow: "0 0 7px rgb(0 0 0 / 90%)",
+          borderRadius: "50%",
+          height: "40px",
+          width: "40px",
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          transition: "box-shadow 0.5s ease",
+          border: "none",
+          "& svg": {
+            "& path": {
+              transition: "stroke 0.5s ease",
+              fill: theme.colors.darkGray,
+            },
+          },
+          "&:hover": { boxShadow: "0 0 12px 2px rgb(0 0 0 / 90%)" },
+          "&:disabled": {
+            boxShadow: "0 0 7px 2px rgb(0 0 0 / 40%)",
+            cursor: "default",
+          },
+        },
+      },
+    },
+    "@keyframes fadeInOut": {
+      "0%": { opacity: 0 },
+      "50%": { opacity: 1 },
+      "100%": { opacity: 0 },
+    },
+    geolocation: {
+      padding: 0,
+      color: theme.colors.darkGray,
+      "&:hover": {
+        color: theme.colors.darkGray,
+      },
```

Comment:
can this color be defined in MapButton ? it seems redundant in fitExtent as well

Reply:
yeah makes sense